2 - RTB/Documentazione esterna/Piano di Progetto/Piano di Progetto.typ

```diff
@@ -656,6 +656,66 @@ Gli obiettivi dell'ottavo sprint riguardano tutti gli aspetti del progetto, di c
 === Diagramma di Gantt
 [da inserire]
 
+
+== Nono Sprint dal 01-01-2024 al 08-01-2024
+
+=== Obiettivi prefissati <obiettivi9>
+Gli obiettivi dell'ottavo sprint si incentrano sul completamento delle attività rimaste sospese nell'ottavo sprint, sulla revisione dei documenti e del PoC in vista della valutazione RTB. In particolare gli obiettivi prefissati per questo sprint sono:
+
+- estensione e revisione del documento Norme di Progetto;
+- correzione degli errori riscontrati in alcuni UC nel documento Analisi dei Requisiti;
```

Comment:
Nella board di Miro, nel To-Do relativo a questo sprint, è segnata anche la task di rigenerazione dei diagrammi relativi all'AdR, che quindi aggiungerei agli obiettivi